### PR TITLE
Show the protection on the portfolio page for Aave v3 mainnet

### PR DESCRIPTION
**Pull request by marcinciarka**
# [Show the protection on the portfolio page for Aave v3 mainnet](https://app.shortcut.com/oazo-apps/story/10245/show-the-protection-on-the-portfolio-page-for-aave-v3-mainnet)
  
## Changes 👷‍♀️
- theres a Protection column in the _new_ owner table
- if position is protected theres a green shield
- if not, theres a button (label/pill) with available action or info (if youre not the owner)
  
## How to test 🧪
- go to your positions

### CODE REVIEW

File: features/vaultsOverview/helpers.tsx [hunk 1]
```diff
@@ -58,20 +58,21 @@ export function getFundingCost({
...
   autoSellData?: AutoBSTriggerData
 }): number {
-  return (
-    stopLossData.stopLossLevel.gt(zero)
-      ? stopLossData.stopLossLevel.times(100)
-      : autoSellData?.execCollRatio.gt(zero)
-      ? autoSellData.execCollRatio
-      : zero
-  ).toNumber()
+  return stopLossData
```
Comment (piekczyk) [inline features/vaultsOverview/helpers.tsx:68]: in that case if stopLossData is missing we will bypass checks for autoSellData and always return zero

Comment (piekczyk) [reply features/vaultsOverview/helpers.tsx:68]: at first glance something like this should do the job 
```
   stopLossData?.stopLossLevel.gt(zero)
      ? stopLossData.stopLossLevel.times(100)
      : autoSellData?.execCollRatio.gt(zero)
      ? autoSellData.execCollRatio
      : zero
```

File: features/vaultsOverview/pipes/positions.ts [hunk 2]
```diff
@@ -79,6 +80,7 @@ export type AavePosition = Position & {
   type: 'borrow' | 'multiply' | 'earn'
   liquidity: BigNumber
   stopLossData?: StopLossTriggerData
+  autoSellData?: AutoBSTriggerData // this is just for type safety in positioins table, its not happening right now
```
Comment (piekczyk) [inline features/vaultsOverview/pipes/positions.ts:83]: typo in `positions`

**Review by piekczyk — CHANGES_REQUESTED** [review]

File: features/vaultsOverview/helpers.tsx (hunk 1 above)
Comment (marcinciarka) [reply features/vaultsOverview/helpers.tsx:68]: Great catch dude.

File: features/vaultsOverview/pipes/positions.ts (hunk 2 above)
Comment (marcinciarka) [reply features/vaultsOverview/pipes/positions.ts:83]: Fixde.

**Review by piekczyk — APPROVED** [review]

**Pull request merged**